src/dsql/ExprNodes.cpp

```diff
@@ -467,6 +467,80 @@ void ArithmeticNode::genBlr(DsqlCompilerScratch* dsqlScratch)
 	GEN_expr(dsqlScratch, arg2);
 }
 
+namespace
+{
+
+const UCHAR f64 = 0;
```

Comment:
@AlexPeshkoff these names seems very common to be in the file namespace, also I think constants should be all upper-cased. I believe some PREFIX_ and upper-case names would be clear.